source/onboard/migrate-from-slack.rst

```diff
@@ -0,0 +1,194 @@
+  - The second command can take a long time if you have a large number of file uploads. If it's interrupted, delete the file generated (if any), and start again.
+
+The file ``export-with-emails-and-attachments.zip`` now contains all the information necessary to be imported into Mattermost.
+
+From the slack advanced exporter repo:
+Due to archive/zip limitations, these actions cannot modify archive in place. It's preferable to fetch e-mails first to avoid copying large attachments around.
+
+.. important::
+
+  Avoid unzipping and rezipping the Slack export. Doing so can modify the directory structure of the archive which could cause issues with the import process.
+
+4. Convert Slack import to Mattermost's bulk export format
+----------------------------------------------------------
+
+Now that you have a Slack export file with emails and attachments, let's convert this information into Mattermost's bulk import format using the import preparation tool ``mmetl``. Download the latest release of ``mmetl`` for your `OS and architecture <https://github.com/mattermost/mmetl/releases/>`__. Run ``mmetl help`` to learn more about using the tool.
+
+Next, run the following command to create a Mattermost bulk import file. Replace ``<TEAM NAME>`` with the name of your team in Mattermost:
+
+.. code:: bash
+
+    ./mmetl transform slack --team <TEAM NAME> --file export-with-emails-and-attachments.zip --output mattermost_import.jsonl
+
+The tool outputs a [.jsonl](https://jsonlines.org/examples) file containing all of your users, channels, and posts. It also creates a ``data`` folder that contains all of your attachments.
+
+.. important::
+
+  It doesn't matter what you name the ``.jsonl`` file. You can name it what you want with the `--output` flag as shown above. It just needs to be a ``.jsonl`` file.
+
+The file ``mattermost-bulk-import.zip`` is now ready to import into Mattermost.
+
+5. Import data into Mattermost
+------------------------------
+
+We have two options to run the import process:
+
+1. SSH into the Mattermost server's host, upload the export file to this server's file system somehow, and use the ``mattermost`` command to process the export file.
+2. Uploading the export through Mattermost's API, via command line ``mmctl`` from the server or from another computer. This option is required for Mattermost Cloud deployments.
+
+Note that for the second option, the server will save the import in its file store before running the import (e.g. AWS S3 if you are using that are your file store), so there will be time spent uploading/downloading the file in this case. Since this is really just a one-time thing to import this file, we recommend the first option of running it on the server itself if you have a large export zip file.
```

Comment:
```suggestion
For the second option, the server will save the import in its file store before running the import (e.g. AWS S3 if you are using that are your file store), so there will be time spent uploading/downloading the file in this case. Since this is really just a one-time thing to import this file, we recommend the first option of running it on the server itself if you have a large export zip file.
```